--- a/clc/Makefile.orig	2012-06-29 11:29:29.000000000 +0000
+++ b/clc/Makefile	2012-06-29 11:30:44.000000000 +0000
@@ -9,11 +9,11 @@
 all: deps build
 test: deps build-test
 
-lib/bcprov.jar:
+ive.xml:
 	@mkdir -p ${TOP}/clc/lib
 	@git clone --depth=1 git://github.com/eucalyptus/eucalyptus-cloud-libs.git lib
 
-deps: lib/bcprov.jar
+deps: ivy.xml
 
 update_properties: ${web_prop_dir}/eucalyptus-web.properties
